static/src/javascripts/__flow__/types/ab-tests.js

```diff
@@ -31,7 +32,8 @@ declare type EpicVariant = Variant & {
     componentName: string,
     template: EpicTemplate,
     classNames: string[],
-    showTicker: boolean,
+    showTicker?: boolean,    // Deprecated, use tickerSettings instead
```

Comment:
Once the relevant changes have been made to the epic admin tool, support for `showTicker` could be removed right? (Just want to make sure I understand correctly!)

Reply:
Yes that's right, I've left it here as an optional field for backwards compatibility